torch/csrc/api/include/torch/nn/functional/embedding.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+namespace torch {
+namespace nn {
+namespace functional {
+
+inline Tensor one_hot(const Tensor& x, int64_t num_classes = -1) {
```

Comment:
nit: naming it `tensor` instead of `x` would be awesome, to match the Python version :)